### ISSUE TITLE
Make `aria-label` of `IconButton` mandatory.

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#2214]
### Description

Make `aria-label` of `IconButton` mandatory.

### Problem Statement/Justification

For accessibility reasons, `aria-label` should be made mandatory.

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by hirotomoyamada** [comment]
@setodeve 

Branches are created from v2.0.

**Comment by setodeve** [comment]
@hirotomoyamada 
Is this issue duplicate with [Issue#1737](https://github.com/yamada-ui/yamada-ui/issues/1737) ?

**Comment by odendayoko** [comment]
@taiseiiiii 
My PR was closed due to a conflict with `v2.0`.
It seems that you are responsible for the accessibility aspect of `v2.0`, so I will reassign this to you.
If you’d like, please refer to my previous PR.
